earth_enterprise/src/third_party/skia/SConscript

```diff
@@ -100,6 +100,8 @@ sgl_env.UpdateCppflagsForSkia()
 
 cxx_flags = sgl_env['CXXFLAGS']
 cxx_flags.remove('-Werror')
+cxx_flags.append('-fPIC')
```

Comment:
Actually, I don't have write access to @tst-eclamar's repo. So, I can't make the change. Eric, can you add this comment? 
```
# Required on some platforms for generating dynamically linked libraries 
# For more info: http://www.microhowto.info/howto/build_a_shared_library_using_gcc.html#idp25536
```